pkg/controller/migplan/validation.go

```diff
@@ -361,6 +363,8 @@ func (r ReconcileMigPlan) validateSourceNamespaces(plan *migapi.MigPlan) error {
 	ns := kapi.Namespace{}
 	notFound := make([]string, 0)
 	for _, name := range namespaces {
+		namespaceMapping := strings.Split(name, ":")
+		name = namespaceMapping[0]
```

Comment:
If namespace name that comes from spec has format of "foo:bar", we should only take "foo" for validation